app/controllers/events_controller.rb

```diff
@@ -5,6 +5,8 @@ class EventsController < ApplicationController
   before_action :authorize_event_author, only: [:update, :destroy]
 
   def index
+    @events = @project.events
+    @project_id = @project.id
```

Comment:
for me mixing concerns was using index action in events controller only to fetch project, not not to use @project instance at all